products/models.py

```diff
@@ -9,88 +9,72 @@ class Category(models.Model):
     class Meta:
         db_table = 'categories'
 
-    def __str__(self):
-        return self.name
-
-
 class Type(models.Model):
     name                = models.CharField(max_length=100)
     thumbnail_image_url = models.URLField(max_length=2000)
-    category            = models.ForeignKey('Category', related_name='category_type', on_delete=models.CASCADE)
+    category            = models.ForeignKey('Category', on_delete=models.CASCADE)
 
     class Meta:
         db_table = 'types'
 
-    def __str__(self):
-        return self.name
-
-
 class Product(TimeStampedModel):
     name                  = models.CharField(max_length=500)
     price                 = models.DecimalField(max_digits=9, decimal_places=2)
     description           = models.TextField()
     thumbnail_image_url   = models.URLField(max_length=2000)
-    type                  = models.ForeignKey('Type', related_name='product_type', on_delete=models.CASCADE)
+    type                  = models.ForeignKey('Type', on_delete=models.CASCADE)
+    tag                   = models.ManyToManyField('Tag',through='TagProduct')
```

Comment:
`ManyToManyField`는 컬럼명을 복수형으로 작성해주세요.
```suggestion
    tags                  = models.ManyToManyField('Tag',through='TagProduct')
```

Reply:
리뷰 감사합니다! 
수정해서 다시 올리겠습니다!